lib/raven/interfaces/http.rb

```diff
@@ -15,6 +15,7 @@ class HttpInterface < Interface
     def initialize(*arguments)
       self.headers = {}
       self.env = {}
+      self.cookies = nil
       super(*arguments)
```

Comment:
This change seems unrelated to the rest of the PR. Perhaps this was accidentally added?